lua/lvim/lsp/peek.lua

```diff
@@ -32,6 +32,10 @@ local function create_floating_file(location, opts)
     math.min(range["end"].line + 1 + (opts.context or 10), range.start.line + (opts.max_height or 15)), -- Don't let the window be more that 15 lines long(height)
     false
   )
+  if next(contents) == nil then
+    print "peek: Unable to get contents of the file!"
```

Comment:
Perhaps use `vim.notify`?

Reply:
there are a couple of more print statements in that file, so we might need to create a general output function that uses vim.notify if available and print as a backup, but that will be in another PR 😅 

cc: @kylo252

Reply:
no, users can disable it if they want
```lua
lvim.builtin.notify.active = false
```

Reply:
But vim.nortify is a built-in function, notify plugin just overrides it?

Reply:
yes you are right on that front, ok let me replace all of those print statements with vim.notify